images/runtime/commonbase/buster.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian:buster-slim
```

Comment:
we can use oryx-buster-base also instead of adding a new tag. IMO tagging will add little more complexity to our template and scripts, but I'm open to better ideas